aws/resource_aws_cloudtrail_test.go

```diff
@@ -441,6 +442,48 @@ func testAccAWSCloudTrail_event_selector(t *testing.T) {
 	})
 }
 
+func testAccAWSCloudTrail_event_selector_delete(t *testing.T) {
```

Comment:
```suggestion
func TestAccAWSCloudTrail_eventSelectorDelete(t *testing.T) {
```
The capitalization and naming in this file are odd. Most that I've seen have exported tests - the test itself is a capitalized function. I'm surprised at this working and running. This is also not the standard Go convention. Normally you wouldn't have an underscore in the name.  But, the Terraform (sort of) standard I've seen it is the resource/acceptance test prefix in camel case, then underscore, then the specific issue being tested:

Let's look at an example, `TestAccAWSIAMRole_testNameChange`. Here's what's going on:
* `TestAccAWS` - prefix for all AWS acceptance tests
* `IAMRole` - resource being tested
* `_` - underscore between test name and prefix (not Go convention)
* `testNameChange` - what aspect is being tested (probably didn't need "test" here)

Reply:
The lowercase test naming in this file is due to serializing all the tests for this resource. See the addition on line 30.

Reply:
@spavuluri Definitely listen to @bflad over me!

@bflad I hadn't seen tests serialized like that yet. It makes perfect sense why they aren't exported. However, there's still a bit of tech debt there: a combination of camel case and underscored test names. I defer to you if this is not the right place to fix. Otherwise, I'd recommend these names:

```
testAccAWSCloudTrail_basic
testAccAWSCloudTrail_cloudwatch
testAccAWSCloudTrail_enableLogging
testAccAWSCloudTrail_includeGlobalServiceEvents
testAccAWSCloudTrail_isMultiRegion
testAccAWSCloudTrail_isOrganization
testAccAWSCloudTrail_logValidation
testAccAWSCloudTrail_kmsKey
testAccAWSCloudTrail_tags
testAccAWSCloudTrail_eventSelector
```